### PR TITLE
[Doc] Update audit_loader.md

### PR DESCRIPTION
**Pull request by WuMenglong**
修改审计插件文档中部分表述，更新建表为最新版兼容插件的语句

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

**Review by EsoragotoSpirit — DISMISSED** [review]

File: docs/en/administration/audit_loader.md [hunk 1]
```diff
@@ -16,198 +16,50 @@ Because the fields of audit logs vary among different StarRocks versions, you mu
 
 > **CAUTION**
 >
-> DO NOT change the table schema in the examples, or the log loading will fail.
-
-- StarRocks v2.4, v2.5, v3.0, v3.1, and later minor versions:
+> - DO NOT change the table schema in the examples, or the log loading will fail.
+> - Because the fields of audit logs vary among different StarRocks versions, the new version AuditLoader collects the common fields admong them from all available StarRocks versions currently.
```
Comment (evelynzhaojie) [inline docs/en/administration/audit_loader.md:20]: admong  --- among

Comment (evelynzhaojie) [reply docs/en/administration/audit_loader.md:20]: from all available StarRocks versions currently --- from all available StarRocks versions

File: docs/en/administration/audit_loader.md [hunk 2]
```diff
@@ -16,198 +16,50 @@ Because the fields of audit logs vary among different StarRocks versions, you mu
...
+  `planMemCosts`      DOUBLE                     COMMENT "Memory cost for planning in bytes"
 ) ENGINE = OLAP
 DUPLICATE KEY (`queryId`, `timestamp`, `queryType`)
 COMMENT "Audit log table"
 PARTITION BY RANGE (`timestamp`) ()
 DISTRIBUTED BY HASH (`queryId`) BUCKETS 3 
 PROPERTIES (
   "dynamic_partition.time_unit" = "DAY",
-  "dynamic_partition.start" = "-30",
+  "dynamic_partition.start" = "-30",       -- Keep the audit logs from the latest 30 days. You can adjust this value based on you bussiness demand.
```
Comment (evelynzhaojie) [inline docs/en/administration/audit_loader.md:57]: bussiness --- business

File: docs/en/administration/audit_loader.md [hunk 3]
```diff
@@ -16,198 +16,50 @@ Because the fields of audit logs vary among different StarRocks versions, you mu
...
+  `scanBytes`         BIGINT                     COMMENT "Size of the scanned data in bytes",
+  `scanRows`          BIGINT                     COMMENT "Row count of the scanned data",
+  `returnRows`        BIGINT                     COMMENT "Row count of the result",
+  `cpuCostNs`         BIGINT                     COMMENT "CPU resources consumption time for query in nanoseconds",
+  `memCostBytes`      BIGINT                     COMMENT "Memory cost for query in bytes",
+  `stmtId`            INT                        COMMENT "Incremental SQL statement ID",
+  `isQuery`           TINYINT                    COMMENT "If the SQL is a query (0 and 1)",
+  `feIp`              VARCHAR(128)               COMMENT "IP address of FE that executes the SQL",
+  `stmt`              VARCHAR(1048576)           COMMENT "Original SQL statement",
+  `digest`            VARCHAR(32)                COMMENT "SLOW SQL fingerprint",
```
Comment (evelynzhaojie) [inline docs/en/administration/audit_loader.md:47]: Slow

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Pull request merged**